### PR TITLE
[WIP]Text coloring when there is a file:line in the output

### PR DESCRIPTION
**Pull request by sierrafayad**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Fixed tickets | #336

This will add coloring to the File:line output, as we discuss in the related issue #336 I picked a combination of colors at based on the 'severity' of the categories.

### CODE REVIEW

File: src/Application/Console/Formatters/Console.php [hunk 1]
```diff
@@ -541,7 +541,20 @@ private function getFileLinkFormatter(): FileLinkFormatter
...
+    {
+        //black, red, green, yellow, blue, magenta, cyan, white, default
+        $categoryColor = [
+            'Code' => 'cyan',
+            'Complexity' => 'green',
+            'Architecture' => 'blue',
+            'Style' => 'yellow',
+            'Security' => 'red',
+        ];
+        return $categoryColor[$category] ?? '';
```
Comment (olivernybroe) [inline src/Application/Console/Formatters/Console.php]: Just to be sure, so if the category does not exist we return an empty string as the color. 
Does that actually work, will that result in no coloring or a crash?

Comment (sierrafayad) [reply src/Application/Console/Formatters/Console.php]: I think I left that in there to set the default color, I should have added a TODO 😅
Lets say we end up choosing blue for any unknown category 
```suggestion
        return $categoryColor[$category] ?? 'blue';
```

Comment (olivernybroe) [reply src/Application/Console/Formatters/Console.php]: Sounds perfect, let's do that.

**Comment by olivernybroe** [comment]
@Jibbarth No breaking changes here, should we make a minor release with this change or take it together with the slevomat/php7.4 upgrade?

File: src/Application/Console/Formatters/Console.php (hunk 1 above)
Comment (sierrafayad) [reply src/Application/Console/Formatters/Console.php]: I had to commit it I think

**Comment by Jibbarth** [comment]
@olivernybroe As it's new feature, we should pass to `1.12` version. 

I think it would be better if we bring slevomat/php7.4 and other features with that :wink:

**Comment by olivernybroe** [comment]
@Jibbarth Alright, sounds like a plan 😄

**Comment by nunomaduro** [comment]
@Jibbarth Can you a tweet or let me know once this is out?

**Comment by nunomaduro** [comment]
Just made it: https://twitter.com/enunomaduro/status/1219196179398975488.

**Pull request merged**